app/domain/output_document.rb

```diff
@@ -33,15 +33,15 @@ def guider_organisation
   end
 
   def value_of_pension_pots
-    return 'No value given' if appointment_summary.value_of_pension_pots.blank?
-
-    pension_pot = number_to_currency(appointment_summary.value_of_pension_pots, precision: 0)
-
-    if appointment_summary.upper_value_of_pension_pots.blank?
-      pension_pot
-    else
-      pension_pot + ' to ' + \
-        number_to_currency(appointment_summary.upper_value_of_pension_pots, precision: 0)
+    pension_pot = to_currency(appointment_summary.value_of_pension_pots)
+    upper_limit = to_currency(appointment_summary.upper_value_of_pension_pots)
```

Comment:
I've updated the approximate variable. Tried changing the others but it ends up a bit harder to parse the method (you've got to read the whole thing quite carefully when pension_pot is referred to as `lower_limit` or `lower_value`).